### PR TITLE
Update T/T Affects

### PR DESCRIPTION
**Pull request by sniper7kills**
`affects-mails` Needs an example of the configuration & Usecases. (I'll probably play with it later as that's something I've yet to use)

I think in this PR; I'll also make an architecture page to briefly describe the system as a whole linking to each piece for additional information.

Question that needs to be considered: Is there a need for creating an `affects-custom` page for describing how to create a custom affect?

### CODE REVIEW

File: docs/tenancy/1.x/affects-cache.md [hunk 1]
```diff
@@ -9,19 +9,38 @@ tags:
...
+1. [Overview](#overview)
+2. [Installation](#installation)
+3. [Configuration](#configuration)
+    1. [Example](#example)
+
+## Overview
+
+**Purpose**
+
+The purpose of this package is to allow the use of different broadcast drivers and settings based on the tenant.
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/affects-cache.md:23]: Says `broadcast` but should be `cache drivers`

File: docs/tenancy/1.x/affects-filesystems.md [hunk 2]
```diff
@@ -9,19 +9,39 @@ tags:
...
+1. [Overview](#overview)
+2. [Installation](#installation)
+3. [Configuration](#configuration)
+    1. [Example](#example)
+
+## Overview
+
+**Purpose**
+
+The purpose of this package is to allow the separation of tenant files
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/affects-filesystems.md:23]: files, directories or even storage servers

File: docs/tenancy/1.x/affects-logs.md [hunk 3]
```diff
@@ -9,19 +9,39 @@ tags:
...
+1. [Overview](#overview)
+2. [Installation](#installation)
+3. [Configuration](#configuration)
+    1. [Example](#example)
+
+## Overview
+
+**Purpose**
+
+The purpose of this package is to allow the logging of errors into a separate file or location for each tenant.
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/affects-logs.md:23]: Or different services.

**Comment by sniper7kills** [comment]
affects-models example updated; untested, but I believe should provide a more universal example of changing the connection used by 3rd party packages.

Needs review

**Comment by sniper7kills** [comment]
affects-mails has been updated to include configuration instructions and a simple example. (untested)

File: docs/tenancy/1.x/affects-mails.md [hunk 4]
```diff
@@ -9,13 +9,56 @@ tags:
...
+```php
+namespace App\Listeners;
+
+use Tenancy\Affects\Mails\Events\ConfigureMails;
+
+class ConfigureTenantMails
+{
+    public function handle(ConfigureMails $event)
+    {
+        $event->setFrom([$event->event->tenant->email_from => $event->event->tenant->email_name]);
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/affects-mails.md:61]: Wrap this in a if statement to check if there's a tenant :)

File: docs/tenancy/1.x/affects-urls.md [hunk 5]
```diff
@@ -9,22 +9,40 @@ tags:
...
+1. [Overview](#overview)
+2. [Installation](#installation)
+3. [Configuration](#configuration)
+    1. [Example](#example)
+
+## Overview
+
+**Purpose**
+
+The purpose of this package is to modify domain being used when generating URL's
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/affects-urls.md:23]: Dot at the end.

File: docs/tenancy/1.x/affects-models.md [hunk 6]
```diff
@@ -9,31 +9,51 @@ tags:
...
+1. [Overview](#overview)
+2. [Installation](#installation)
+3. [Configuration](#configuration)
+    1. [Example](#example)
+
+## Overview
+
+**Purpose**
+
+The purpose of this package is to allow the modification of models provided by other packages
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/affects-models.md:23]: Dot at the end

File: docs/tenancy/1.x/affects-configs.md [hunk 7]
```diff
@@ -9,20 +9,39 @@ tags:
...
+1. [Overview](#overview)
+2. [Installation](#installation)
+3. [Configuration](#configuration)
+    1. [Example](#example)
+
+## Overview
+
+**Purpose**
+
+The purpose of this package is to allow the use change the configuration for different integrations or packages based on the current tenant
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/affects-configs.md:23]: Dot at the end

File: docs/tenancy/1.x/architecture.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+## Lifecycle Hooks
+These components focus on what happens when a tenant is created, updated, or deleted in the system.
+Things as simple as ensuring that the domain a tenant sign up with is valid, to creating databases and registering/creating services to handle the needs to the tenant occur in these components.
+
+[Read More])(hooks-general)
+
+## Affects
+The Affect compoents are responsible for changing the application for the tenant that has been identified.
+Setting up connections to databases, changing logging settings, and the configuration of other services
+occur in these components.
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/architecture.md:30]: Can you doublecheck this order in comparison with the documentation and make sure they are in the same order?

**Pull request merged**